addon/templates/components/validated-input.hbs

```diff
@@ -38,18 +38,19 @@
   {{#if (eq type "select")}}
 
     {{one-way-select
-      value            = (or value (get model name))
-      options          = options
-      optionLabelPath  = optionLabelPath
-      optionValuePath  = optionValuePath
-      optionTargetPath = optionTargetPath
-      name             = name
-      class            = config.css.control
-      update           = (action "update")
-      focusOut         = (action "setDirty")
-      includeBlank     = includeBlank
-      disabled         = disabled
-      multiple         = multiple
+    value               = (or value (get model name))
+    options             = options
+    optionLabelPath     = optionLabelPath
+    optionValuePath     = optionValuePath
+    optionTargetPath    = optionTargetPath
+    name                = name
+    class               = "form-control"
+    update              = (action "update")
+    focusOut            = (action "setDirty")
+    prompt              = (if includeBlank (or includeBlank null))
```

Comment:
`prompt` is currently implemented as an alias for `includeBlank` (which is not ideal, see https://github.com/DockYard/ember-one-way-controls/issues/152). Also, i'd prefer this addon to involve as less "magic" as possible when it comes to wrapping one-way-inputs. Since supporting both `prompt` and `includeBlank` seems to be impossible at the moment, i'd suggest just passing
```
prompt = prompt
```